src/vanilla/utils/unwrap.ts

```diff
@@ -95,10 +95,10 @@ export function unwrap<Value, Args extends unknown[], Result, PendingValue>(
       return atom(
         (get) => {
           const state = get(promiseAndValueAtom)
-          if ('v' in state) {
-            return state.v
+          if ('f' in state) {
+            return state.f
```

Comment:
We can add a comment.
```suggestion
          if ('f' in state) { // is pending
            return state.f
```